sectionsUtilityTool/src/main/resources/sectiontool.properties

```diff
@@ -0,0 +1,35 @@
+#/api/v1/accounts/1/terms?per_page=4000&_=1426266199833
+# ^ starting of exp. $ ending of exp
+# ? is a quantifier in regex. hence escaping
+# GET request
+canvas.api.terms=^/api/v1/accounts/1/terms\\?per_page=4000&_=\\d+$
```

Comment:
Agreeing with Zhen's comment: Giving the exact number makes the REGEX fragile.  Checking that the value is a number should be sufficient.